### PR TITLE
Set up a "Hello world" project with two apps

### PR DESCRIPTION
**Pull request by Kumilachew-g**
**In this PR, I have done the following tasks.**
- Create a new Rails API app called 'hello-rails-back-end'.
- Initialized my project with Git.
- Checked my project has a Postgres database set up.
- Created a table for storing your messages and creating 5 different greetings.
- Created an API endpoint that selects a random greeting from your table (you will need a controller with an action on the rails route).

### CODE REVIEW

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @Kumilachew-g , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] hi as part of the requirements, you are supposed to create a table that stores your greetings and add 5 greetings to it but you don't have it in your backend. ill recommend adding it to your `seed.rb` file. by doing so it will automatically add the greetings to the database when we set up the database

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kumilachew-g** [comment]
Hi @m05e5, I have done it. Thanks, A lot!!!

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+# Build JSON APIs with ease [https://github.com/rails/jbuilder]
+gem 'jbuilder'
+
+# Use Redis adapter to run Action Cable in production
+# gem "redis", "~> 4.0"
+
+# Use Kredis to get higher-level data types in Redis [https://github.com/rails/kredis]
+# gem "kredis"
+
+gem 'rack-cors'
```
Comment (TedLivist) [inline Gemfile:36]: - [ ] **[OPTIONAL]** I had issues running your app due to the CORS policy issue. See the error I get.

  fheu

  You have done well uncommenting the `rack-cors` gem but you still need to create `config/initializers/cors.rb` file and add this code to it:

  ```
  Rails.application.config.middleware.insert_before 0, Rack::Cors do
    allow do
      origins '*'
      resource '*', headers: :any, methods: [:get, :post, :patch, :put]
    end
  end
  ```

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Kumilachew-g,

✔️ Good documentation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Gemfile (hunk 1 above)
Comment (Kumilachew-g) [reply Gemfile:36]: Fixed, Thanks @TedLivist!!!

**Comment by Eldonratzlaff** [comment]
Hello
I think this code review is not for me

**Pull request merged**